### PR TITLE
LPS-151362 | Tests Automation Add an Alert in context when segmentation is disabled

### PR DESCRIPTION
**Pull request by rafaellajordao**
JIRA Ticket:
[LPS-151362](https://issues.liferay.com/browse/LPS-151362)
↳ [LPS-156782](https://issues.liferay.com/browse/LPS-156782)
Motivation:
Add an Alert in context when segmentation is disabled

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
> ###  ci:test:tango - 164 out of 179 jobs passed in 2 hours 34 minutes
> Click here for more details.

these failures have nothing to do with the test automation :)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisable.testcase [hunk 1]
```diff
@@ -524,4 +524,150 @@ definition {
...
+		}
+	}
+
+	@description = "LPS-151362: Given segmentation is disabled at Instance Settings
+	When user navigates to Experience simulation sidepanel
+	Then the warning alert explains how to re-enable it"
+	@priority = "3"
+	test ViewReEnabledAlertAtExperienceSimulationSidepanel {
+		task ("Add segment") {
+			Segmentation.openSegmentsAdmin(siteURLKey = "guest");
```
Comment (Luizcarlosqueiroz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisable.testcase:629]: Can you modify this block to use JSONWS? Like this example:
```
JSONSegmentsentry.addSegment(
	fieldName = "First Name",
	groupName = "Test Site Name",
	operator = "equals",
	segmentName = "First Segment",
	text = "Test");
```

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationDisable.testcase:629]: ok!!

**Comment by rafaellajordao** [comment]
ci:test:tango

**Comment by rafaellajordao** [comment]
ci:forward

**Pull request closed without merging**